packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -75,6 +75,8 @@ export class BrowserPlatform implements Platform {
     }
 
     const v = new Uint8Array(nBytes);
+    // Polyfill for IE11.
+    const crypto = window.crypto || (window as any).msCrypto;
```

Comment:
You probably need to suppress eslint here to pass tests.

Reply:
Done.

Reply:
It seems like you could do this once at the top of the file (after imports before the class).